app/controllers/api/v1/admin/organizations_controller.rb

```diff
@@ -0,0 +1,41 @@
+module Api
+  module V1
+    module Admin
```

Comment:
(Question): I don't feel confident that I understand why we have these API routes under `/admin`? 

i.e. `PUT /api/admin/organizations/{:id}`  instead of PUT `/api/organizations/{:id}`.

Reply:
I think it came from me starting out creating and placing controller files and routes near where I saw similar divisions around `users`, but there's no reason they couldn't all live in one namespace/ route pattern and be differentiated by  the :before_actions in the controller.